client/src/components/Footer.js

```diff
@@ -1,41 +1,45 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import HomeIcon from '../assets/icons/Icon-home.svg';
-import ListIcon from '../assets/icons/Icon-list.svg';
-import CreateIcon from '../assets/icons/Icon-plus-circle.svg';
+import PropTypes from 'prop-types';
 
 const FooterContainer = styled.div`
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
   width: 100%;
+  height: 50px;
 `;
 
-const FooterIcon = styled.input`
+const FooterIcon = styled.button`
+  display: flex;
+  justify-content: center;
+  min-width: 100px;
   cursor: pointer;
   border: none;
   &:focus {
     outline: none;
   }
 `;
 
-export const Footer = () => (
```

Comment:
Actually, it isn't a footer, even if it appears at the bottom :-D  I would rename it to bottom navigation, tab navigation or something like this. 
Material UI calls it even bottom navigation:  https://material-ui.com/components/bottom-navigation/

Reply:
Thanks, I will change it! 😄